lib/manager/bundler/update-locked.ts

```diff
@@ -0,0 +1,23 @@
+import { logger } from '../../logger';
+import { UpdateLockedConfig, UpdateLockedResult } from '../types';
+import { extractLockFileEntries } from './locked-version';
+
+export function updateLockedDependency(
+  config: UpdateLockedConfig
+): UpdateLockedResult {
+  const { depName, currentVersion, newVersion, lockFile, lockFileContent } =
+    config;
+  logger.debug(
+    `bundler.updateLockedDependency: ${depName}@${currentVersion} -> ${newVersion} [${lockFile}]`
```

Comment:
should we use logger metadata here?

Reply:
I think this line is ok, we already have it for others